WordPress/Classes/ViewRelated/Blog/Blog Dashboard/ViewModel/BlogDashboardViewModel.swift

```diff
@@ -2,8 +2,18 @@ import Foundation
 import UIKit
 import CoreData
 
-typealias DashboardSnapshot = NSDiffableDataSourceSnapshot<DashboardCardSection, DashboardCardModel>
-typealias DashboardDataSource = UICollectionViewDiffableDataSource<DashboardCardSection, DashboardCardModel>
+enum DashboardSection: CaseIterable {
```

Comment:
[Q] Should this be Hashable?

Reply:
It is Hashable actually, but there's no need to declare it.

From [Apple's documentation](https://developer.apple.com/documentation/swift/hashable):
> An enum without associated values has Hashable conformance even without the declaration.